torch/_inductor/kernel/bmm.py

```diff
@@ -87,6 +89,24 @@ def bmm_grid(b, m, n, meta):
 
 @register_lowering(aten.bmm)
 def tuned_bmm(mat1, mat2, *, layout=None):
+    # Make the inputs of bmm contiguous
+    # because bmm cpu implementation does contiguous() if not
+    # this is to avoid additional copies in bmm
```

Comment:
Is this comment accurate? If the input is transposed in the last two dims, would bmm still make the inputs contiguous?

Reply:
Thanks and modified!